src/libraries/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj

```diff
@@ -152,7 +152,9 @@
     <Compile Include="Design\DesignerTransactionTests.cs" />
     <Compile Include="Security\Authentication\ExtendedProtection\ExtendedProtectionPolicyTypeConverterTests.cs" />
     <Compile Include="XTypeDescriptionProviderTests.cs" />
-
+    <!-- The collection definitions must be in the same assembly as the test that uses them -->
```

Comment:
I would probably skip the comment duplication. It seems like with this change there will be enough examples to follow and missing reference will result in clear build failure. And there is explanation in the DisableParallelization.cs